### ISSUE TITLE
Random acid grenades in security crates

### ISSUE THREAD

**Issue by Hoshi-chan** [d3athrow/vgstation13#18479]
When did this happen, how did this happen, and most importantly, why did this happen?

If cargo is busting open weapon crates, they're violating space law and prone to having their shit pushed in by either Security or RoboCop borgs. It creates drama and interaction between players, and has always been a part of the game. Cargo gets belligerent and arms up, they get raided, that's the way things go.

Discuss.

**Comment by Reviire** [comment]
Retarded

**Comment by Irradiation** [comment]
**The acid smoke effect only exists for the Tactical Assault Gear crate.**
Ackshually, it's sulphuric acid and capsain or whatever pepperspray has in it. The sacid smoke funtime only triggers upon "non-standard" crate opening methods (Read: Emag or an ID with valid access). When the PR for Tacticool Assault Gear crates got merged, I believe we pulled a post from Pomf or something saying that crates shouldn't be booby-trapped for cargonians or scishitters. Yes, it's fucking stupid and yes it should be gone.
